specification/kubernetesconfiguration/resource-manager/readme.go.md

```diff
@@ -0,0 +1,19 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go)
+go:
+  license-header: MICROSOFT_APACHE_NO_VERSION
+  clear-output-folder: true
+```
+
+### Tag: package-2019-11-01-preview and go
+
+These settings apply only when `--tag=package-2019-11-01-preview --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2019-11-01-preview' && $(go)
+namespace: Microsoft.Azure.Management.KubernetesConfiguration
```

Comment:
A go package name should only contain lower case letters and numbers. Could you please change this? Maybe something like this:
```suggestion
namespace: kubernetesconfiguration
```
it depends on your service context (which I do not have much knowledge), your choice

Reply:
Fixed